apps/schematic/api/docker-compose.yml

```diff
@@ -12,8 +15,9 @@ services:
     #   - ./docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
     networks:
       - schematic
-    ports:
-      - "${SERVER_PORT}:7080"
+    ports: 
+      - "${SERVER_PORT}:80"
+      - "443:443"
```

Comment:
The monorepo aims to standardize how ports are defined In order to prevent ports from clashing.

- Schematic ports should be 7xxx
- The port exposed should be defined in `devcontainer.json`

Reply:
Based on conversation with @tschaffter , port 80 is also protected. Maybe tried to configure http portocol on 8080 (or anything above 1024)

Reply:
@tschaffter marked this as resolved for now. I have updated nginx to use port 8080 for http request instead of port 80.